saseumn/views/users.py

```diff
@@ -159,6 +159,8 @@ def resumes(id=None):
         if not os.path.exists(path):
             return abort(404)
         return send_file(path, attachment_filename=resume.name)
+    if request.method == "POST":
+        db.session.query(Resume).filter(Resume.id==request.form.get("delete")).delete()
```

Comment:
shouldn't this use the `DELETE` http method instead of `POST` if ur deleting resumes

Reply:
Apparently you can't use DELETE without Javascript? And I'm too lazy lmaoooo